pkgs/by-name/re/redocly/package.nix

```
@@ -32,24 +32,27 @@ buildNpmPackage rec {
 
     mkdir $out/bin
     makeWrapper $out/lib/node_modules/@redocly/cli/node_modules/@redocly/cli/bin/cli.js \
-      $out/bin/redocly-cli \
+      $out/bin/redocly \
       --set-default REDOCLY_TELEMETRY off \
       --set-default CI true # Silence update messages
+
+    # Symlink for backwards compatibility. Remove after deprecation.
+    ln -s $out/bin/redocly $out/bin/redocly-cli
   '';
 
   installCheckPhase = ''
     runHook preInstallCheck
-    $out/bin/redocly-cli --version
+    $out/bin/redocly --version
     runHook postInstallCheck
   '';
 
   doInstallCheck = true;
 
   meta = {
-    description = "Redocly CLI makes OpenAPI easy. Lint/validate to any standard, generate beautiful docs, and more.";
+    description = "Redocly CLI makes OpenAPI easy. Lint/validate to any standard, generate beautiful docs, and more";
```

Don't repeat package name.

```suggestion
    description = "A tool that makes OpenAPI easy. Lint/validate to any standard, generate beautiful docs, and more";
```